eZ/Publish/API/Repository/Tests/URLAliasServiceTest.php

```diff
@@ -1600,6 +1600,52 @@ public function testDeleteCorruptedUrlAliases()
         );
     }
 
+    public function testRenamingParentContentDoesntBreakChildAlias(): void
+    {
+        $repository = $this->getRepository();
+        $locationService = $repository->getLocationService();
+        $urlAliasService = $repository->getURLAliasService();
+        $contentService = $repository->getContentService();
+
+        $languageCode = 'eng-GB';
+
+        // 1. Create parent folder
+        $folder = $this->createFolder([$languageCode => 'a'], 2);
+        $folderLocationId = $folder->contentInfo->getMainLocationId();
+
+        // 2. Create child folder
+        $child = $this->createFolder([$languageCode => 'b'], $folderLocationId);
+        $childLocationId = $child->contentInfo->getMainLocationId();
+        $childLocation = $locationService->loadLocation($childLocationId);
```

Comment:
Side note: you don't need to load that directly:
```suggestion
        $childLocation = $child->getVersionInfo()->getContentInfo()->getMainLocation()
        $childLocationId = $childLocation->id; 
```
(or use later on `$childLocation->id` directly).